recipes/restbed/all/conandata.yml

```diff
@@ -0,0 +1,12 @@
+sources:
+  "4.8":
+    url: "https://github.com/Corvusoft/restbed/archive/refs/tags/4.8.tar.gz"
+    sha256: "4801833f86a67b8a123c2c01203e259eb81157e1e9ef144a3b6395cb2d838a42"
+patches:
```

Comment:
This project is active, might be worth tossing up a PR 😉

Reply:
https://github.com/Corvusoft/restbed/pull/532